extensions/amp-carousel/amp-carousel.md

```diff
@@ -47,7 +47,7 @@ The carousel consists of an arbitrary number of items, as well as optional navig
 
 The carousel advances between items if the user swipes, uses arrow keys, or clicks an optional navigation arrow.
 
-**Example**: While the example shows a carousel of images, `amp-carousel` supports arbitrary children.
+**Example**: While the example shows a carousel of images, `amp-carousel` supports arbitrary HTML children.
```

Comment:
I would refrain using callouts for examples, otherwise, our docs will be all callouts. :)